website/src/components/layout/navigation/navigation.js

```diff
@@ -2,31 +2,17 @@
 import { Fragment } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import gql from 'graphql-tag';
 
 import { jsx, useBrand } from '@westpac/core';
 import { Heading } from '@westpac/heading';
-import { useQuery } from '@apollo/react-hooks';
 import { NavigationBlock } from './navigation-block';
 
-export const Navigation = ({ components }) => {
-	const { SPACING } = useBrand();
+// root paths that don't need to be dynamic
```

Comment:
Added this PATHS array for root pages like home and tokens. This is assuming that all other items will have children.

Reply:
The alternative would have been to add a field in the settings with the correct path. This was avoided as the settings JSON file is already not the easiest to edit and work with in the admin ui. We also wanted to avoid making routing even more of a responsibility for authors.